### PR TITLE
Push Chat Implementation - Fork Merge Request

### PR DESCRIPTION
**Pull request by muktanshumishra24**

### CODE REVIEW

File: src/components/PushChat/PushChat.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Chat } from "@pushprotocol/uiweb";
+import { useAccount } from "wagmi";
+
+
+const PushChat = () => {
+  const { address } = useAccount()
+  console.log(process.env.REACT_APP_PUSH_CHAT_API_KEY)
+  return (
+      <Chat
+        account = {address!}//user address
```
[the remark this answers is not in the document]
Comment (Xand6r) [reply src/components/PushChat/PushChat.tsx:10]: @aiswaryawalter

[the remark this answers is not in the document]
Comment (muktanshumishra24) [reply src/components/PushChat/PushChat.tsx:10]: Discussed and Solved

File: src/components/PushChat/PushChat.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        supportAddress="0x9Fb3d86157a9e2dC2a771C297f88FA9784fa4e31" //support address
```
[the remark this answers is not in the document]
Comment (muktanshumishra24) [reply src/components/PushChat/PushChat.tsx:11]: Done

File: src/components/PushChat/PushChat.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        apiKey={process.env.REACT_APP_PUSH_CHAT_API_KEY}
```
[the remark this answers is not in the document]
Comment (muktanshumishra24) [reply src/components/PushChat/PushChat.tsx:12]: Done

**Pull request closed without merging**